modules/core/shared/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -186,4 +191,34 @@ object DynamicConfig {
       GmosSouth(GmosCommonDynamicConfig.Default, None, None, None)
   }
 
+  import GnirsConfig.GnirsFpu
+
+  final case class Gnirs(
+    camera:               GnirsCamera,
+    decker:               GnirsDecker,
+    disperser:            GnirsDisperser,
+    disperserOrder:       GnirsDisperserOrder,
+    exposureTime:         Duration,
+    filter:               GnirsFilter,
+    fpu:                  GnirsFpu,
+    prism:                GnirsPrism,
+    readMode:             GnirsReadMode,
+    wavelengthSuggestion: GnirsWavelengthSuggestion
+  ) extends DynamicConfig.Impl(Instrument.Gnirs)
+
+  object Gnirs {
+    val Default: Gnirs = Gnirs(
+      GnirsCamera.ShortBlue,
+      GnirsDecker.Acquisition,
+      GnirsDisperser.D32,
+      GnirsDisperserOrder.Three,
+      java.time.Duration.ZERO,
+      GnirsFilter.Order5,
+      GnirsFpu.Slit(GnirsFpuSlit.LongSlit4),
```

Comment:
Perhaps we should pick a name like LongSlit_0_30 that corresponds, somewhat, to the display name?  Or do they naturally think of the FPU as LongSlit4 instead of 0.30 arcsec?

Reply:
In SQL for `short_name` and `long_name` the 0.30 arsec name is used. LongSlit4 is only used for the Id, and therefore in code. GMOS and F2 follow the same naming convention. 

If we want to change this convention I could open a ticket take care of renaming the slits for all instruments later.

Reply:
When the science staff writes a query to find observations that use that FPU, it would probably be better to match on the id LongSlit_0_30 instead of LongSlit4 I'm guessing.